stories/Tables/Tables.stories.tsx

```diff
@@ -0,0 +1,641 @@
+import React from 'react';
+
+export const TableBase = () => (
+  <table className='table'>
+    <thead>
+      <tr>
+        <th scope='col'>#</th>
+        <th scope='col'>Nome</th>
+        <th scope='col'>Cognome</th>
+        <th scope='col'>Username</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope='row'>1</th>
+        <td>Mario</td>
+        <td>Verdi</td>
+        <td>mario.verdi</td>
+      </tr>
+      <tr>
+        <th scope='row'>2</th>
+        <td>Francesco</td>
+        <td>Bianchi</td>
+        <td>francesco.bianchi</td>
+      </tr>
+      <tr>
+        <th scope='row'>3</th>
+        <td>Alessandro</td>
+        <td>Rossi</td>
+        <td>alessandro.rossi</td>
+      </tr>
+    </tbody>
+  </table>
+);
+
+export const TableDark = () => (
+  <table className='table table-dark'>
+    <thead>
+      <tr>
+        <th scope='col'>#</th>
+        <th scope='col'>Nome</th>
+        <th scope='col'>Cognome</th>
+        <th scope='col'>Username</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope='row'>1</th>
+        <td>Mario</td>
+        <td>Verdi</td>
+        <td>mario.verdi</td>
+      </tr>
+      <tr>
+        <th scope='row'>2</th>
+        <td>Francesco</td>
+        <td>Bianchi</td>
+        <td>francesco.bianchi</td>
+      </tr>
+      <tr>
+        <th scope='row'>3</th>
+        <td>Alessandro</td>
+        <td>Rossi</td>
+        <td>alessandro.rossi</td>
+      </tr>
+    </tbody>
+  </table>
+);
+
+export const TableThead = () => (
+  <>
```

Comment:
Why this `Fragment` here?

Reply:
Because there is two children, it's not right?

Reply:
Yes, just saw it 👍